examples/react/src/App.js

```diff
@@ -39,6 +46,14 @@ let eosTransitWalletProviders = [
   // }),
 ];
 
+let ualProviders = [
```

Comment:
Are these commented out because they are redundant with eos-transit?
One of the goals of the sample app is to allow us to test eos-auth using ghost-inspector, so we'd want these working ideally. Is there an easy way to enable both eos-transit and UAL providers in the same code? If not, it's not worth a lot of effort here, but just wondering

Reply:
Yes, right now we can only handle 1 of each type. Adding `eos-transit-scatter` and `ual-scatter` at the same time will throw an error warning them to remove one of them.

If we wanted to allow both I think we would try 1 and then try the other if that fails? The additional provider would be more of a backup.